src/__tests__/app.test.ts

```diff
@@ -52,6 +58,12 @@ beforeAll(async () => {
   );
   server = http.createServer(app);
   server.listen();
+  gracefulShutdown(server, {
```

Comment:
You are not testing [this code](https://github.com/teamdigitale/io-backend/pull/431/files#diff-916156c481141bfcd08c6ae40ce10e00R71), you are testing something else here, a similar, but different code - if somebody changes the code in `server.ts`, this will not be affected.

Moreover, you aren't testing anything here, how are you making sure the server waits for the active connections to complete before shutting down?